### PR TITLE
Rename CallCallbackError to InvokeCallbackError in `sixtyfps::interpr…

### PR DESCRIPTION
**Pull request by tronical**
…eter`

cc #431

### CODE REVIEW

File: api/sixtyfps-rs/migration.md [hunk 1]
```diff
@@ -59,3 +59,4 @@ fn model_tracker(&self) -> &dyn ModelTracker {
 ### Interpreter
 
 `Value::Array` was removed and `Value::Model` needs to be used instead.
+`CallCallbackError` was renamed to `InvokeCallbackError`.
```
Comment (ogoffart) [inline api/sixtyfps-rs/migration.md:62]: Ah well, technically the docs for the interpreter should go in the interpreter's doc.

**Review by ogoffart — APPROVED** [review]

File: api/sixtyfps-rs/migration.md (hunk 1 above)
Comment (tronical) [reply api/sixtyfps-rs/migration.md:62]: I don't quite understand. `InvokeCallbackError` is an Rust interpreter specific API that is here mentioned in the interpreter specific section.

Comment (tronical) [reply api/sixtyfps-rs/migration.md:62]: Ohh, I see, this is the migration doc for the Rust API create, not for the interpreter. Hmm.

Comment (tronical) [reply api/sixtyfps-rs/migration.md:62]: Let's tackle it when we integrate `migration.md` into the actual docs (not linked yet) -> #888

**Pull request merged**